tests/ltp/patches/fix_sigrelse_sigrelse01.patch

```diff
@@ -0,0 +1,673 @@
+This test case creates a child process using "FORK_OR_VFORK".
+Fork behavior is not supported in sgx-lkl.
+Hence, the below modifications are performed.
+	1. Removed the code related to child process creation.
+	2. Moved child and parent-related code/functionality
+	   to a single function "verify_sigrelse"
+	3. Signal with ID 34 is skipped from testing.
+	   This signal ID is used for internal signals in
+	   the thread library of sgx-lkl-musl.
+	4. Removed pipe handling code which is used for
+	   interprocess communication between parent and child.
+diff --git a/testcases/kernel/syscalls/sigrelse/sigrelse01.c b/testcases/kernel/syscalls/sigrelse/sigrelse01.c
+index 38086caa7..61915f008 100644
+--- a/testcases/kernel/syscalls/sigrelse/sigrelse01.c
++++ b/testcases/kernel/syscalls/sigrelse/sigrelse01.c
+@@ -116,23 +116,17 @@ extern int sighold(int __sig);
+ extern int sigrelse(int __sig);
+ #endif
+ 
+-/* Needed for NPTL */
++// Needed for musl pthread implementation
+ #define SIGCANCEL 32
+ #define SIGTIMER 33
++#define SIGSYNCCALL 34
+ 
+ void setup(void);
+ void cleanup(void);
+-static void parent(void);
+-static void child(void);
+-static void timeout(int sig);
++static void verify_sigrelse(void);
+ static int setup_sigs(void);
+ static void handler(int sig);
+ static void wait_a_while(void);
+-static char *read_pipe(int fd);
+-static int write_pipe(int fd, char *msg);
+-static int set_timeout(void);
+-static void clear_timeout(void);
+-static void getout(void);
+ int choose_sig(int sig);
+ 
+ #define TRUE  1
+@@ -141,29 +135,12 @@ int choose_sig(int sig);
+ #ifndef DEBUG
+ #define DEBUG 0
+ #endif
+-
+-#define CHILD_EXIT(VAL) ((VAL >> 8) & 0377)	/* exit value of child process */
```

Comment:
I think this is now dead code, please leave it in: eventually we will want to revert these and it makes it much easier to understand the diffs if they only include functional changes.

Reply:
This patch file will be removed/reverted once #598 is fixed.